README.md

```diff
@@ -1,74 +1,150 @@
 # ob-operator
 
-ob-operator 可以让 OceanBase 以容器的方式，无缝运行在公有云或私有部署的 Kubernetes 集群上。
+The ob-operator is a Kubernetes operator that simplifies the deployment and management of OceanBase cluster and related resources on Kubernetes.
 
-ob-operator 现已支持 OceanBase 集群的创建、删除，完整的节点生命周期管理，并通过 Service 的形式暴露给用户使用。后续会支持租户管理、ob-proxy 管理、多 Kubernetes 集群等功能。
+## Quick Start
 
-目前支持通过 YAML、Kustomize 来部署 ob-operator。后续会支持 Helm 形式部署。
+This section provides a step-by-step guide on how to quickly use ob-operator for managing OceanBase, using examples of deploying ob-operator and deploying an OceanBase cluster.
 
-![01](docs/img/01.png)
+### Prerequisites
 
-## 环境依赖
+Before getting started, please ensure you have a functional Kubernetes cluster with at least 2 CPU cores, 10GB of memory, and 100GB of storage space. 
```

Comment:
we need to emphasize  the amount of resource is available, so it's better to add an available at the end